utils/oc.py

```diff
@@ -529,12 +538,18 @@ def init_oc_client(self, cluster_info):
                 jump_host = cluster_info.get('jumpHost')
             else:
                 jump_host = None
-            self.set_oc(
-                cluster,
-                OC(server_url, token, jump_host,
-                   settings=self.settings,
-                   init_projects=self.init_projects,
-                   init_api_resources=self.init_api_resources))
+            try:
+                oc_client = OC(server_url, token, jump_host,
+                               settings=self.settings,
+                               init_projects=self.init_projects,
+                               init_api_resources=self.init_api_resources)
+                self.set_oc(cluster, oc_client)
+            except StatusCodeError as e:
+                logging.error('The cluster is unreachable: error %s', str(e))
```

Comment:
two log messages? I believe this one can be removed.

Reply:
sure thing.  I did like how it listed the full server address but cluster name is sufficient.

Reply:
you can still have it, I'm only asking for de-duplication. content is up to you.